code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -16,6 +16,8 @@
 	var/silent = FALSE
 	var/weapon_type
 	var/weapon_name_simple
+	/// Does having this weapon out reduce the chemical recharge rate of the changeling?
```

Comment:
Could probably reword it to something like "how much the recharge rate is reduced by having this weapon out", since its not a boolean.

Reply:
Good catch!